app/code/Magento/Checkout/view/frontend/web/js/view/billing-address.js

```diff
@@ -127,11 +128,13 @@ function (
          */
         useShippingAddress: function () {
             if (this.isAddressSameAsShipping()) {
+                window.isbothAddressSame = true;
                 selectBillingAddress(quote.shippingAddress());
 
                 this.updateAddresses();
                 this.isAddressDetailsVisible(true);
             } else {
+                window.isbothAddressSame = false;
```

Comment:
Hi @Dharmeshvaja91. Use global variables is bad practice. Please create local variable or set it to the class context.